rpcs3/Crypto/unself.cpp

```diff
@@ -1316,7 +1326,7 @@ bool CheckDebugSelf(const std::string& self, const std::string& elf)
 
 		// Copy the data.
 		char buf[2048];
-		while (ssize_t size = s.read(buf, 2048))
+		while (size_t size = s.read(buf, 2048)) // Did size need to be of type ssize_t?
```

Comment:
Though I'm not sure, it doesn't look good. Errors are returned as -1 https://github.com/RPCS3/rpcs3/blob/b77666cbbad44edcb992f49168d6b617c2a4152d/Utilities/File.cpp#L693 (even though return type is unsigned) and some code may rely on that. Maybe @Nekotekina or @Bigpet will know for sure.